src/Neha/LargestNum.java

```diff
@@ -0,0 +1,30 @@
+//Find the largest of three numbers using Logical And operator.
+package Neha;
+
+import java.util.Scanner;
+
+public class LargestNum {
+	static void largestNum(int x, int y, int z) {
```

Comment:
this simple we can check if already know that z is small than why compare that value again with x and y..
correct that logic.

if( n1>=n2 && n1>=n3)
        println("lf is the largest number."+ n1);
    else if (n2>=n1 && n2>=n3)
        println("lf is the largest number."+ n2);
    else
        println("lf is the largest number."+ n3);